config/transform-elasticsearch/documents/datadiscovery_germplasm.json

```diff
@@ -4,8 +4,12 @@
 
   "document-transform": {
     "@type": [ "Germplasm" ],
+    "entryType": [ "Germplasm" ],
     "@id": "{.germplasmPUI}",
+    "identifier": "{.germplasmPUI}",
+    "name": "{.germplasmName}",
 
+    "accessionHolder": "{.holdingInstitute.instituteName}",
```

Comment:
That field should not be useful in WheatIS/DataDiscovery since it is related to RARe flavor to link to the rare-basket app.